scripts/bump.sh

```diff
@@ -141,17 +142,16 @@ log_info 'Bumping `backend`'
 	[ -z "$DRY_RUN" ] && npm version "${NEW_VERSION}" --no-git-tag-version
 )
 
-log_info 'Commiting `frontend` and `backend` bump'
-[ -z "$DRY_RUN" ] && { git commit -m "Bump frontend and backend versions to ${NEW_VERSION}" || exit 1; }
-
 # ----------------------------- Bump root and push -----------------------------
 
 log_info 'Bumping `root`'
-[ -z "$DRY_RUN" ] && { npm version "${NEW_VERSION}" -m "Bump to version %s" || exit 1; }
+[ -z "$DRY_RUN" ] && { npm version "${NEW_VERSION}" --no-git-tag-version || exit 1; }
+[ -z "$DRY_RUN" ] && { git commit -m "Bump to version ${NEW_VERSION}" || exit 1; }
```

Comment:
is this the commit?

'cause it only looks like it moved, not it was removed :thinking:

Reply:
No it didn't just moved! The first commit has been removed [here](https://github.com/iScsc/iscsc.fr/pull/70/commits/a39a1034d56926c17bbcea14f74c92a017fd61be) and the 2nd was automatically made by `npm version` --> I added `--no-git-tag-version` to every `npm version` call, then the only commit now is the `git commit` at line 149

Reply:
aaah

`npm version ... -m ...` does a `git` commit?
that would explain the "disappearing commit" :+1:

Reply:
Exactly!